app/scripts/early_voting_mgr.js

```diff
@@ -25,7 +25,15 @@ define(
     function whenMarkerEventsHappen(eventType, marker) {
       if (eventType === 'click') {
         for (var i = 0; i < earlyVotingLocations.length; i++) {
+
           if (marker.getPosition().equals(earlyVotingLocations[i].getGeometry().get())) {
+            
+            var collapsedToggle = Boolean(document.getElementById('#collapse-toggle'+i).getAttribute("aria-expanded"));
+
+            if (collapsedToggle) {
+              $('#collapse'+i).collapse('show');       
```

Comment:
how did this make its way into multiple pull requests :P we might not want to do this, based off some of the design stuff that @WheresHJ was showing, maybe we can talk about this on tuesday?